ipa-metrics/src/key.rs

```diff
@@ -114,8 +114,8 @@ impl<'lv, const LABELS: usize> Name<'lv, LABELS> {
 /// This is the key inside metric stores which are simple hashmaps.
 #[derive(Debug, Clone, Eq)]
 pub struct OwnedName {
-    key: &'static str,
-    labels: [Option<OwnedLabel>; 5],
+    pub key: &'static str,
+    pub labels: [Option<OwnedLabel>; 5],
```

Comment:
I haven't dug in to where this gets used, but exposing the padded array as a public interface seems like it has some potential for confusion.

Reply:
you're right and I don't think it is even necessary given that `labels()` method exists